src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -23,6 +26,9 @@ namespace Stratis.Bitcoin.Features.Wallet
     /// </summary>
     public class WalletManager : IWalletManager
     {
+        // <summary>As per RPC method definition this should be the max allowable expiry duration.</summary>
+        private const int maxDurationInSeconds = 1073741824;
```

Comment:
Maybe clearer name would be maxUnlockDurationInSec or maxSecretCacheDurationInSec

Reply:
👍